LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2014 Katrina Owen, _@kytrinyx.com
+Copyright (c) 2017 Exercism, Inc
```

Comment:
The `(c)` form of **&#169;** is used when the typsetting equipment does not have it available, and is even then not necessarily recognized legally as indicating copyright notice.  We have the typesetting available and should use the officially/legally recognized symbol, or `copr.` or `copyright`.

My suggestion then is to in the word, and leave out the `(c)`.

In case of concerns for ASCII characters, this is character `0169`.